### DISCUSSION TITLE
Do I misunderstand this question as well?

### DISCUSSION THREAD

**Discussion by hannibalmads** [TabularEditor/TabularEditor3#1259] [Learn]
![image](https://github.com/TabularEditor/TabularEditor3/assets/116729399/a3c23069-008c-4ba1-b3c3-baefc9bbb07c)

In my opinion the table has 6 dependencies since the "indirectly column" is also a "directly column". On that basis I answered 6, since there are 6 unique columns that the table depends on.

**Comment by mlonsk** [comment]
Hi @hannibalmads 
Sorry for the delay in responding, we are looking into the question formulation and will get back to you.
